clifford/tools/g3c/__init__.py

```diff
@@ -560,14 +625,14 @@ def val_truncated_get_line_reflection_matrix(line_array, n_power):
     mat2solve = np.zeros((32, 32), dtype=np.float64)
     for i in range(line_array.shape[0]):
         LiMat = get_left_gmt_matrix(line_array[i, :])
-        tmat = np.matmul(np.matmul(LiMat, mask_2minus4), LiMat)
+        tmat = (LiMat@mask_2minus4)@LiMat
         mat2solve += tmat
-    mat_val = np.matmul(mask1, mat2solve)
+    mat_val = mask1@mat2solve
     mat_val = mat_val[1:6, 1:6].copy()/line_array.shape[0]
     if n_power != 1:
         c_pow = 1
         while c_pow < n_power:
-            mat_val = np.matmul(mat_val, mat_val)
+            mat_val = mat_val@mat_val
             c_pow = c_pow*2
     return mat_val
```

Comment:
Would be a good use case for `np.linalg.matrix_power`, but I guess numba doesn't support it.